### ISSUE TITLE
oauth 2.0.1 vs oauth 2.1 on QBQ

### ISSUE THREAD

**Issue by sparrell** [sFractal-Podii/quizquadaminos#1202]
2.1 works on local environments but doesn't work on deployments. Why? Fix

**Comment by TraceyOnim** [comment]
The issue is with OTP 26  when trying to verify the SSL during the Oauth flow, here is a related issue https://community.fly.io/t/error-when-verifying-the-ssl-of-my-website-during-oauth-flow/16173.

I would suggest we maintain the `oauth 2.0.1` version since it does not have an issue with OTP 26. Updating to the latest  version will require us to downgrade to OTP 25.

**Comment by sparrell** [comment]
Agree. But let's keep this task open and lower priority watching for when newer versions of either come out to see if it's fixed and we can upgrade.
